css/properties/field-sizing.json

```diff
@@ -4,7 +4,7 @@
       "field-sizing": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/field-sizing",
-          "spec_url": "https://drafts.csswg.org/css-ui/#field-sizing",
+          "spec_url": "https://drafts.csswg.org/css-forms-1/#propdef-field-sizing",
```

Comment:
Looks like this feature moved from one module to another.

http://drafts.csswg.org/css-ui-3/ doesn't seem to have it, see: https://raw.githubusercontent.com/w3c/webref/main/ed/ids/css-ui-3.json